README.md

```diff
@@ -1,2 +1,149 @@
-# jupyter-itsecl
-ECL jupyter kernel implementation with typescript based on jupyter-jpecl-kernel
+# ITSECL
+
+ITSECL is a Typescript/ECL kernel for the [Jupyter notebook](http://jupyter.org/)...
+It provide a prototype implementation of HPCC ECL for Jupyter Notebook by referencing [ITypeScript](https://github.com/nearbydelta/itypescript)
+
+The HPCC Systems server platform is a free, open source, massively scalable platform for big data analytics. Download the HPCC Systems server platform now and take the reins of the same core technology that LexisNexis has used for over a decade to analyze massive data sets for its customers in industry, law enforcement, government, and science.
+
+For more information and related downloads for HPCC Systems Products, please visit
+https://hpccsystems.com
+
+
+## Installations
+
+Prerequites:
+python-dev, python3.5-dev, python-pip python3-pip jupyter nodejs
+
+For example on Ubuntu 16.04 xenial
+```sh
+sudo apt-get install -y curl git python-dev python3.5-dev python3-pip
+sudo pip3 install jupyter
+sudo curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install -y nodejs
+
+```
+On ubuntu 18.04 replace python3.5-dev with python3.6-dev
+
+Install ITSECL
+### Install ITSECL globally
+You don't need use "sudo" but usually required to configure npm default directory
```

Comment:
You don't need to use "sudo" but it is usually required to configure the npm default directory